charts/primary-site/values.yaml

```diff
@@ -1,3 +1,8 @@
 globals:
   site_token:
   foxglove_api_url: https://api.foxglove.dev
+
+primary_site:
```

Comment:
using "primary_site" as a name/identifier in this feels odd to me. I think the term is useful for contrast with an edge site, but this is the chart for a primary site, so the user has this context already. I would probably just remove primary_site, but maybe add the "lake" nesting roman notes below.

Taking https://github.com/bitnami/charts/blob/main/bitnami/postgresql-ha/values.yaml as an example, they have "global" as a top-level key, not "postgres". I don't know how much of the inbox listener/stream service/garbage collector stuff we are looking to expose to the user, but to me it would also make sense to break those into their own stanzas - so something like,

```
global:
  tag: abc
inboxListener:
  ...
  deployment:
    resources:
      cpu:
        requests:
        limits:
      memory:
        requests:
        limits:

garbageCollector:
   ...
   deployment:
    ...
streamService:
  ...
  deployment:
    resources:
      ...
```

etc. Anything that is remotely parameterizable in the templates, needs to be parameterizable from the values file IMO.

Reply:
another thought -
I think putting an "env" section under each of these and populating the env vars directly that way is a little more transparent than what we have here. When we put "lakeStorageProvider" as a top-level key, or even a key under "lake", we are sort of inventing a new config scheme that differs from the actual config scheme. If we can get away with not doing that, it seems preferable IMO. Ideally the real config scheme is sane enough that direct mapping isn't a burden.

Reply:
meh, feel less good about this "env" idea. Seems like there probably isn't a reason to expose the fact that these are getting translated to env vars by the template, so just having config options under the stanzas (but separate from "resources") seems better

Reply:
I like this format especially when it comes to configuring the _deployment_ options.